tools/release.sh

```diff
@@ -2,7 +2,7 @@
 
 LANGUAGE=$1
 
-VERSION=$(grep -Po "\d{1,}(\.\d{1,}){1,2}$" main_${LANGUAGE}.tex)
+VERSION=$(grep -Eo "\d{1,}(\.\d{1,}){1,2}$" main_${LANGUAGE}.tex)
```

Comment:
Let me guess, you're on MacOS?

Reply:
In any case, with `-E` this regexp doesn't work. Please restore `-P` or come up with something else.

Reply:
This should also work on Linux, could you test it?

```
echo "Version 1.2.3 is the best" | grep -Eo '[0-9]+(\.[0-9]+){1,2}'
```

Reply:
Also I'm not sure parsing the version for `tex` file is the right way. Rather it should be the opposite - specifying the version during build and injecting it to the document

Reply:
Sure, I have some spare time this week. I can send a POC to showcase the idea and we can discuss on PR.

Are there any specific requirements for the release? What is the flow? Run the release script and upload the pdfs manually?

Reply:
@qwrtln can we close this topic and have it merged?

Reply:
Sorry, busy day. Yes, looks good now.

Reply:
> Sure, I have some spare time this week. I can send a POC to showcase the idea and we can discuss on PR.
> 
> Are there any specific requirements for the release? What is the flow? Run the release script and upload the pdfs manually?

This is no software, we have no SLAs :-) Whenever we feel like it's time to release, we try to freeze any changes and let the translators finish their work. Then, after they're done, we just build all the files locally and attach them to the GitHub release.